src/frontend/app.ts

```diff
@@ -140,6 +142,35 @@ export default (octokitApp: App): Express => {
 		)
 	);
 
+	app.get(["/session", "/session/*"], (req: Request, res: Response, next:NextFunction) => {
```

Comment:
New session redirect page where it saves the `jiraHost` as a cookie from `localStorage` value saved.